tycho-targetplatform/src/main/java/org/eclipse/tycho/targetplatform/TargetDefinition.java

```diff
@@ -69,6 +69,11 @@ public interface Location {
 	public interface InstallableUnitLocation extends Location {
 
 		public static String TYPE = "InstallableUnit";
+		
+		/**
+		 * Must match {@link org.eclipse.tycho.core.TargetPlatformConfiguration#referencedRepositoryMode}
+		 */
+		public static boolean FOLLOW_REPOSITORY_REFERENCES_DEFAULT = true;
```

Comment:
@laeubi In Tycho 4 `TargetPlatformConfiguration#referencedRepositoryMode` defaults to ignore. In PDE, it defaults to `include`. In Tycho5, it also defaults to `include`. Should I keep `true` here and change the behavior in Tycho 4, or should I make it `false` to maintain backward compatibility for Tycho 4 but remain inconsistent with PDE?

Reply:
The TargetPlatform Configuration default should be false as it can influence other places as well (e.g. when using pom defined repository). For the Target Platform Location we can use `true` as default.

Just in case, for the "IncludeSourceMode" we even have multiple options that is either:

- ignore --> always ignore regardless of what is configured in the target file
- honor --> use whats configured in the target file
- force --> always enable regardless whats configured in the target file